dids/src/main/kotlin/web5/sdk/dids/DidResolutionResult.kt

```diff
@@ -71,29 +71,4 @@ public class DidResolutionMetadata(
   public var additionalProperties: MutableMap<String, Any>? = null,
 )
 
-/**
- * Contains metadata about the DID document.
- *
- * @property created Timestamp of when the DID was created.
- * @property updated Timestamp of the last time the DID was updated.
- * @property deactivated Indicates whether the DID has been deactivated. `true` if deactivated, `false` otherwise.
- * @property versionId Specific version of the DID document.
- * @property nextUpdate Timestamp of the next expected update of the DID document.
- * @property nextVersionId The version ID expected for the next version of the DID document.
- * @property equivalentId Alternative ID that can be used interchangeably with the canonical DID.
- * @property canonicalId The canonical ID of the DID as per method-specific rules.
- * @property types Returns types for DIDs that support type indexing.
- */
-public class DidDocumentMetadata(
```

Comment:
moved to [here](https://github.com/TBD54566975/web5-kt/pull/248/files#diff-32048dcbb8e15e4b662972033bb8904c81001a1a76ee44f5d835a901bf816121R1)